db/db_test_util.h

```diff
@@ -49,10 +49,7 @@
 #include "util/mutexlock.h"
 
 #include "util/string_util.h"
-// SyncPoint is not supported in Released Windows Mode.
```

Comment:
Right.. I don't know why we have the comment saying it not being support in release mode..